components/Pagination.js

```diff
@@ -112,7 +113,8 @@ class Pagination extends Component {
       ...other
     } = this.props;
 
-    const { page, pageSize } = this.state;
+    const statePage = this.state.page;
+    const statePageSize = this.state.pageSize;
```

Comment:
Is there a reason for setting these variables? Why can't you just say `this.state.page` when you need it. 

```js
const statePage = this.state.page;
const statePageSize = this.state.pageSize;
```